.pubnub.yml

```diff
@@ -1359,69 +1371,78 @@ sdks:
                   minimum-os-version:
                     - iOS 9.0
                   maximum-os-version:
-                    - iOS 14.4
+                    - iOS 14.6
                   target-architecture:
                     - arm64
                     - armv7
                   target-devices:
                     - iPhone
                     - iPad
-                tvOS:
+          - distribution-type: library
+            distribution-repository: GitHub relese
+            package-name: PubNub.ios.xcframework.tar.gz
+            location: https://github.com/pubnub/objective-c/releases/download/v4.17.0/PubNub.ios.xcframework.tar.gz
+            supported-platforms:
+              supported-operating-systems:
+                iOS:
                   runtime-version:
                     - Objective-C 2.0
                   minimum-os-version:
-                    - tvOS 10.0
+                    - iOS 9.0
                   maximum-os-version:
-                    - tvOS 14.4
+                    - iOS 14.6
                   target-architecture:
                     - arm64
+                    - armv7
                   target-devices:
-                    - Apple TV
-                watchOS:
-                  runtime-version:
-                    - Objective-C 2.0
-                  minimum-os-version:
-                    - watchOS 4.0
-                  maximum-os-version:
-                    - watchOS 7.3
-                  target-architecture:
-                    - armv7k
-                    - arm64_32
-                  target-devices:
-                    - Apple Watch
-          - distribution-type: framework
-            distribution-repository: GitHub release
-            package-name: PubNub.catalyst.xcframework.tar.gz
-            location: https://github.com/pubnub/objective-c/releases/download/v4.17.0/PubNub.catalyst.xcframework.tar.gz
+                    - iPhone
+                    - iPad
+          - distribution-type: library
+            distribution-repository: GitHub relese
+            package-name: PubNub.macos.framework.tar.gz
+            location: https://github.com/pubnub/objective-c/releases/download/v4.17.0/PubNub.macos.framework.tar.gz
             supported-platforms:
               supported-operating-systems:
                 macOS:
                   runtime-version:
                     - Objective-C 2.0
                   minimum-os-version:
-                    - macOS 11.0
+                    - OS X 10.11
                   maximum-os-version:
-                    - macOS 11.2
+                    - macOS 11.4
                   target-architecture:
                     - arm64
                     - x86_64
-                iOS:
+          - distribution-type: library
+            distribution-repository: GitHub relese
```

Comment:
Can you please fix the typo here and same ones in the rest of the file. Rest looks good.